packages/apps/policy-send-counter/src/lib/vincent-policy.ts

```diff
@@ -242,11 +242,11 @@ export const vincentPolicy = createVincentPolicy({
         `[@lit-protocol/vincent-policy-send-counter-limit/commit] Calling incrementByAddress(${ethAddress}) on contract ${counterSignatures.address}`,
       );
 
-      const provider = await Lit.Actions.getRpcUrl({ chain: 'yellowstone' });
-
       // Call contract directly without Lit.Actions.runOnce wrapper
       const txHash = await laUtils.transaction.handler.contractCall({
-        provider,
+        provider: new ethers.providers.JsonRpcProvider(
+          await Lit.Actions.getRpcUrl({ chain: 'yellowstone' }),
+        ),
```

Comment:
Added the correct type in [my current scaffold sdk PR](https://github.com/LIT-Protocol/vincent-scaffold-sdk/pull/8/files#diff-084ddf5cd2dcdf184cad3af453b48443798e8c6fc93ab354d95681d013653dfa). We would have caught this
Great finding!